packages/@aws-cdk/aws-glue-alpha/lib/storage-parameter.ts

```diff
@@ -0,0 +1,323 @@
+/**
+ * The compression type used by `StorageParameters.COMPRESSION_TYPE`.
+ */
+export enum CompressionType {
+  /**
+   * No compression.
+   */
+  NONE = 'none',
+
+  /**
+   * Burrows-Wheeler compression.
+   */
+  BZIP2 = 'bzip2',
+
+  /**
+   * Deflate compression.
+   */
+  GZIP = 'gzip',
+
+  /**
+   * Compression algorithm focused on high compression and decompression speeds, rather than the maximum possible compression.
+   */
+  SNAPPY = 'snappy',
+}
+
+/**
+ * Specifies the action to perform when query results contain invalid UTF-8 character values.
+ */
+export enum InvalidCharHandlingAction {
```

Comment:
These possible values are defined within the `@see` I mentioned within the jsdoc, however I understand this may not be apparent. Should I add a `@see` to this enum also?

Reply:
Gotcha. Yes please include these note into the enum as well.

Reply:
I've added a `@see`